.github/workflows/nightly_tests.yml

```diff
@@ -0,0 +1,47 @@
+name: nightly_tests
+on:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
+  upstream_tests:
+     if: ${{ github.repository == 'md-raid-utilities/mdadm' }}
+     runs-on: self-hosted
+     timeout-minutes: 120
```

Comment:
Shoudn't it be aligned to same value as in `tests.yml`? if it executes same test suite, can we have execution in one yml to avoid such differences?